docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,133 @@
+        #. The exams IDA waffle flag can be found/created in the LMS' Django Admin under "Waffle Flag Course Overrides"
+        #. If needed, you can make a new entry using "course_apps.exams_ida" as the waffle flag, the course ID, and by setting it to "Enabled"
+    #. Find or create two exam subsections with a type of 'Proctored exam' in your test course
+        #. This can be done by creating a subsection and clicking the 'Configure' button (should look like a gear), going to the "Advanced Tab", and selecting "Proctored"
+        #. You MUST also set this subsection to be Graded as a Midterm or Final exam
+        #. This subsection should contain at least one unit with one gradeable block, e.g. a generic multiple choice question in order to test downstream effects to the gradebook
+    #. You should also have a tab open with your event bus logs
+
+
+Proctored Exam Flow
+===================
+Expected Behavior: "A paid track learner is able to start, complete, and submit a proctored exam using Proctoring Software that is launched via LTI."
+
+Can't Start Exam Without Proctoring Software Installed
+------------------------------------------------------
+#. Enter a Proctored Exam without proctoring software installed. You should see an interstitial stating that "This exam is proctored" with a button that says "Continue to Proctored Exam"
+#. Clicking "Continue to Proctored Exam" should bring you to an interstitial titled "Set up and start your proctored exam." with "Start System Check" and "Start Exam" buttons
+#. Clicking the "Start Exam" button should pop up a modal that says "Cannot Start Exam"
+#. Clicking "OK" should close that modal
+#. Clicking "Start System Check" should redirect you to your proctoring setup page
+#. Navigating back one page and clicking "Start Exam" should still block you from starting the exam
+
+Can't Start Exam Without Proctoring Software Running
+----------------------------------------------------
+#. Install the proctoring software that you've configured with your platform. Go through the opening interstitials and click "Start System Check". This should load your respective Proctoring Software Setup page
+#. Immediately navigating back and trying to click "Start Exam" should't allow you to start the exam
+#. Navigating back to your proctoring software setup page and refreshing on that page should cause no issues
+#. Go through your proctoring software setup for your Webcam, Desktop, etc. Upon setup completion, you should be redirected back to an exam interstitial with just the "Start Exam" button
+#. Close your proctoring software and click "Start Exam". The exam should not start
+
+Exam Flow Works With Proctoring Software Installed & Running
+------------------------------------------------------------
+#. Go through the setup again, this time without closing your proctoring software. Clicking "Start Exam" should start the exam
```

Comment:
What setup? The steps in "Can't Start Exam Without Proctoring Software Running"?

Reply:
Yes :)